smart_proxy_pulp.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('webmock', '~> 1')
   gem.add_development_dependency('rack-test', '~> 0')
   gem.add_development_dependency('rake', '~> 10')
+  gem.add_dependency('sys-filesystem', '~> 1.1')
```

Comment:
:+1: df is simple enough instead of calling out to an external ffi gem.
granted it will not work on windows, but since the scope of pulp is only redhat family I think I'm fine with it.